scripts/e2etests/NuGetFunctionalTestUtils.ps1

```diff
@@ -169,12 +169,17 @@ function RealTimeLogResults
 
     if ($currentTestTime -gt $EachTestTimoutInSecs)
     {
+        $logLastLine = (Get-Content $log)[-1] -split " "
+        $currentTestName = $logLastLine[2].Replace("...", "") 
+        $resultRow = "Failed $currentTestName 600000 Test timed out"
+        $resultRow >> $RealTimeResultsFile
         $errorMessage = 'Run Failed - Results.html did not get created. ' `
         + 'This indicates that the tests did not finish running. It could be that the VS crashed or a test timed out. Please investigate.'
         CopyResultsToCI $NuGetDropPath $RunCounter $testResults
         if($env:CI)
         {
-            Get-ScreenCapture -OfWindow -OutputPath $env:EndToEndResultsDropPath
+            # Running into some hangs, so comment it out for now.
+            #Get-ScreenCapture -OfWindow -OutputPath $env:EndToEndResultsDropPath
```

Comment:
Should this entire if block be removed, or commented out?

Reply:
@emgarten  fixed